src/common/rvh/Util.js

```diff
@@ -112,6 +112,27 @@ RvH.common.Util = {
 	},
 
 	/**
+	 * If there is no regex in the link, make it clickable
+	 */
+	 parseAsLink: function(text, host) {
```

Comment:
I think this could be simplified using regular expressions, something like this could work:

```javascript
return text
  .replace(/((Disallow|Allow|Sitemap):\s+)(\/[^\s\*]+)(\n|$)/ig, '$1<a href="' + host + '$3">$3</a>$4')
  .replace(/(\s|^)(((https?:\/\/)?[\w-]+(\.[\w-]+)+\.?(:\d+)?(\/\S*)?))/gi, '$1<a href="$2">$2</a>');
```

This will also match Sitemap directives and any other full urls